README.md

```diff
@@ -113,12 +111,14 @@ Within the container, build the font:
     $ make -C src/icons
 
 <!--- reference links for badges -->
-[all-contrib]: https://img.shields.io/badge/all_contributors-125-orange.svg?style=flat-square "All Contributors badge"
+[all-contrib]: https://img.shields.io/badge/all_contributors-125-orange.svg "All Contributors badge"
```

Comment:
We have 126 contributors now. In my opinion, this badge is unmaintainable since it is not a dynamic badge. Unless the all-contributors bot or the all contributors project provides this, this badge will be outdated most of the time.

Reply:
Oh true! Hmm... I think it's nice to keep, as a show of appreciation. Perhaps we can find some way to remember every time before we make a release? 'Release checklist', or something like that.

Reply:
That is a good idea. I agree.

For the immediate release ( 1.2 ), there was only one contributor that was added and who did everything that went into the release. We should definitely acknowledge them.

And also, since we are already updating this line, it would make sense to send that update with this change-set.

However, we definitely need to _create_ that Release checklist :smile: